addon/styles/_variables.scss

```diff
@@ -6,6 +6,7 @@
 // Notification base
 // -------------------------
 
+$notification-top: 50px !default;
```

Comment:
Not sure.  I don't even know what it does but I was following the conventions in the file.  Also the `frost-theme` stuff does this as well.